examples/embed/widget.py

```diff
@@ -103,7 +103,7 @@ def update_pyramid(self):
 pop = Population()
 pop.render()
 
-tag = embed.autoload_server(pop.layout)
+tag = embed.autoload_server(session_id=pop.session.id, pop.layout)
```

Comment:
OK that seems fine. It would be valuable to codify these situations/use-cases/guideance somewhere.